### PR TITLE
Migrate to Dart 2

### PR DESCRIPTION
**Pull request by SpencerCornish**
Make changes necessary for Dart 2! @davidmarne

### CODE REVIEW

File: example/todos_class_components/web/index.dart [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import 'dart:html';
 import 'package:wui_builder/wui_builder.dart';
-import 'package:wui_builder_todos/todos.dart';
+import '../lib/todos.dart';
```
Comment (SpencerCornish) [inline example/todos_class_components/web/index.dart:3]: This seems like the wrong approach, but `import 'package:wui_builder_todos/todos.dart';` didn't want to work anymore 🤷‍♂️

File: example/docs/lib/src/components/code_view.dart [hunk 2]
```diff
@@ -34,12 +34,16 @@ class CodeViewContent extends PComponent<CodeViewContentProps> {
 
   @override
   void componentDidMount() {
-    prettyPrint();
+    // TODO: I can't seem to get the prettifier dependency to work at all.
```
Comment (SpencerCornish) [inline example/docs/lib/src/components/code_view.dart:37]: @davidmarne Totally lost on why this dependency is not found... Seems like it should be fine, but who knows.

File: example/todos_class_components/web/index.dart (hunk 1 above)
Comment (davidmarne) [reply example/todos_class_components/web/index.dart:3]: I think its just due to your ide configuration. They often have trouble with repos that have nested libs such as this.

File: example/docs/lib/src/components/code_view.dart (hunk 2 above)
Comment (davidmarne) [reply example/docs/lib/src/components/code_view.dart:37]: again, I think its just due to your ide configuration. They often have trouble with repos that have nested libs such as this.

**Pull request closed without merging**